tests/keras_tests/function_tests/test_get_gptq_config.py

```diff
@@ -56,36 +61,106 @@ def random_datagen_experimental():
 
 
 class TestGetGPTQConfig(unittest.TestCase):
-
-    def test_get_keras_gptq_config(self):
-        qc = QuantizationConfig(QuantizationErrorMethod.MSE,
-                                QuantizationErrorMethod.MSE,
-                                weights_bias_correction=False)  # disable bias correction when working with GPTQ
-        cc = CoreConfig(quantization_config=qc)
-        gptq_configurations = [GradientPTQConfig(1, optimizer=tf.keras.optimizers.RMSprop(),
-                                                 optimizer_rest=tf.keras.optimizers.RMSprop(), train_bias=True,
-                                                 loss=multiple_tensors_mse_loss),
-                               GradientPTQConfig(1, optimizer=tf.keras.optimizers.Adam(),
-                                                 optimizer_rest=tf.keras.optimizers.Adam(), train_bias=True,
-                                                 loss=multiple_tensors_mse_loss),
-                               ]
-
-        gptqv2_configurations = [get_keras_gptq_config(n_epochs=1, optimizer=tf.keras.optimizers.Adam())]
-
-
-
-        for i, gptq_config in enumerate(gptq_configurations):
-            keras_post_training_quantization(in_model=build_model(SHAPE[1:]),
-                                             representative_data_gen=random_datagen,
-                                             n_iter=1,
-                                             quant_config=qc,
-                                             gptq_config=gptq_config)
-
-        for i, gptq_config in enumerate(gptqv2_configurations):
-            keras_gradient_post_training_quantization_experimental(in_model=build_model(SHAPE[1:]),
-                                                                   representative_data_gen=random_datagen_experimental,
-                                                                   core_config=cc,
-                                                                   gptq_config=gptq_config)
+    class TestGetGPTQConfig(unittest.TestCase):
```

Comment:
why is there a class inside a class with the same name?

Reply:
This is a mistake I'm fixing it now